src/modules/univ3/logic.swap-token.ts

```diff
@@ -8,13 +8,13 @@ import {
 } from './types';
 import { Currency, CurrencyAmount, Token, TradeType } from '@uniswap/sdk-core';
 import { FeeAmount, Pool, Route, SwapQuoter, computePoolAddress, encodeRouteToPath } from '@uniswap/v3-sdk';
-import { ISwapRouter } from './contracts/SwapRouter';
+import { IV3SwapRouter } from './contracts/SwapRouter02';
 import JSBI from 'jsbi';
-import { Pool__factory, SwapRouter__factory } from './contracts';
+import { Pool__factory, SwapRouter02__factory } from './contracts';
 import { UNSUPPORTED_TOKEN_ERROR } from './errors';
 import * as common from '@protocolink/common';
 import * as core from '@protocolink/core';
-import { getDeadline, toUniToken } from './utils';
```

Comment:
We can remove unused files and codes such as `getDeadline`, `SwapRouter.json`, `ISwapRouter`, and `SwapRouter__factory`.